tests/ports/psoc6/board_only_hw/single/pdm_pcm.py

```diff
@@ -0,0 +1,25 @@
+import os
```

Comment:
This test belongs (in a later stage) to board_ext_hw/multi. So why not putting it there?
Maybe you can generate the test output from the same board (single), but I think that is trickier

Reply:
Yes eventually it will go there. Right now since I had not given much thought about testing and rather take it up with read implementation, I added here. But agreed.